.github/workflows/issue-mark-duplicate.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: mark-duplicate
-        uses: actions-cool/issues-helper@v1.8
+        uses: actions-cool/issues-helper@release
```

Comment:
I was thinking about a `1.x` branch instead. Otherwise be might start using a backwards incompatible `2.x`.